### PR TITLE
Proofread 20200326 - index.md

### PR DESCRIPTION
**Pull request by lscheske**
-----
[View rendered activities/codebase-stewardship/supporting-governance/index.md](https://github.com/publiccodenet/about/blob/Proofreading-20200326/activities/codebase-stewardship/supporting-governance/index.md)

### CODE REVIEW

File: activities/codebase-stewardship/supporting-governance/index.md [hunk 1]
```diff
@@ -2,62 +2,63 @@
...
-  * If so, how is it decided who to gets to take part in the different decision processes
+* How should decisions (small and large) about the community and the codebase be made?
+  * Can certain issues only be decided by certain community members?
+  * Can others be distributed and/or delegated to the wider community or specialists within the community?
+    * If so, how does designation of community members to different decision processes take place?
 
 ## Designing a governance model
 
-After the needs of the community have been established, a suitable governance model can be designed. As a help to get started [this template](governance-template.md) can be used as inspiration.
+After the community's governance needs have been established, a suitable governance model can be designed. To get started, [this template](governance-template.md) can be used as inspiration.
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/supporting-governance/index.md]:
```suggestion
After the community's governance needs have been established, a suitable governance model can be designed. [This governance template](governance-template.md) can be used as inspiration.
```

File: activities/codebase-stewardship/supporting-governance/index.md [hunk 2]
```diff
@@ -2,62 +2,63 @@
...
-Whatever [methods](../../workshops/) are used to figure this out, there are a few basic questions that need to be answered:
+Several basic questions need to be answered regardless of the [methods](../../workshops/) used to determine these needs:
 
-* How do you want to make decisions, large and small, around the codebase and the community?
-  * Can certain issues only be decided by certain community members and can others be distributed and/or delegated to the wider community or specialist parts of the community?
-  * If so, how is it decided who to gets to take part in the different decision processes
+* How should decisions (small and large) about the community and the codebase be made?
+  * Can certain issues only be decided by certain community members?
+  * Can others be distributed and/or delegated to the wider community or specialists within the community?
+    * If so, how does designation of community members to different decision processes take place?
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/supporting-governance/index.md]:
```suggestion
  * Who decides which issues are reserved or delegated, and how can community members participate in the decisionmaking process?
```

File: activities/codebase-stewardship/supporting-governance/index.md [hunk 3]
```diff
@@ -2,62 +2,63 @@
...
 
 ## Anti-patterns
 
 Some common [anti-patterns](https://en.wikipedia.org/wiki/Anti-pattern) that we have noticed are:
 
-* The governance model is setup to require reviewer from another organization.
-  * If one organization is making 90% of the pull requests, perhaps the other organizations cannot keep up with that pace. If the model is too rigid, merging them timely might become a problem.
-  * If one organization has the majority, or all of the reviewers, but has not dedicated time for them to review code from contributors from other organizations, the back log of pull request to review will grow. If the model is too rigid, merging them timely might become a problem.
+* The governance model is set up to require reviewer(s) from other organization(s).
+  * If one organization is making 90% of the pull requests, the other organizations may not be able to keep up with that pace. If the model is too rigid, merging them timely might become a problem.
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/supporting-governance/index.md]:
```suggestion
  * If one organization makes 90% of the pull requests, the other organizations may not be able to keep up with that pace. If the model is too rigid, timely merging might become a problem.
```

File: activities/codebase-stewardship/supporting-governance/index.md [hunk 4]
```diff
@@ -2,62 +2,63 @@
...
+  * If one organization has the majority (or all) of the reviewers, but has not dedicated time for them to review code contributions from other organizations, the back log of pull request to review will grow. If the model is too rigid, merging them timely might become a problem.
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/supporting-governance/index.md]:
```suggestion
  * If a majority (or all) of the reviewers are part of a single organization that doesn't dedicate time to reviewing code contributions from other organizations, the backlog of pull requests to review will grow. If the model is too rigid, timely merging might become a problem.
```

File: activities/codebase-stewardship/supporting-governance/index.md [hunk 5]
```diff
@@ -2,62 +2,63 @@
...
 * Changes in codebase governance
-* Decision making process
-* Conflict Resolution
+* Decision-making process
+* Conflict resolution
 * Intellectual property
-* Code of Conduct
+* Code of conduct
 
 <!-- All these could possibly have their own sub pages that explain them more in detail. -->
```
Comment (bvhme) [inline activities/codebase-stewardship/supporting-governance/index.md]: I think this is a great opportunity to remove this content debt :)

```suggestion
```

**Comment by bvhme** [comment]
I like it, however a conflict needs to be resolved by picking the best option.

**Comment by bvhme** [comment]
The Travis still has this remar, which you can get from the 'view build' page:

```
activities/codebase-stewardship/supporting-governance/index.md:18: MD009 Trailing spaces
activities/codebase-stewardship/supporting-governance/index.md:19: MD022 Headers should be surrounded by blank lines

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
```

These need to be fixed before this pull request can be merged.

**Pull request merged**